src/recaptcha.js

```diff
@@ -85,9 +96,17 @@ export default class ReCAPTCHA extends React.Component {
   }
 
   explicitRender() {
-    if (this.props.grecaptcha && this.props.grecaptcha.render && this._widgetId === undefined) {
+    let render = this.props.grecaptcha && this.props.grecaptcha.render;
+    if (
+      this.props.grecaptcha &&
+      this.props.grecaptcha.enterprise &&
+      this.props.grecaptcha.enterprise.render
+    ) {
+      render = this.props.grecaptcha.enterprise.render;
```

Comment:
probably don't need this last one:
```suggestion
      this.props.grecaptcha.enterprise
    ) {
      render = this.props.grecaptcha.enterprise.render;
```

... although the original code has this *shrug*